features/admin/blocking_login_for_disabled_administrators.feature

```diff
@@ -7,12 +7,19 @@ Feature: Blocking login for disabled administrators
     Background:
         Given the store operates on a single channel in "United States"
         And there is an administrator "admin@example.com" identified by "sylius"
-        And this administrator account is disabled
 
     @ui @api
     Scenario: Sign in with email and password
-        Given I want to log in
+        Given this administrator account is disabled
+        And I want to log in
         When I specify the username as "admin@example.com"
         And I specify the password as "sylius"
         And I log in
```

Comment:
```suggestion
        And I try to log in
```

Reply:
It is available only in the shop, refactoring the scenario is not a subject of this PR so I'd prefer to left it as-is :v.